### PR TITLE
WFP-2619 basic logging endpoint for contact work

### PR DESCRIPTION
**Pull request by sambailey-moj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/controller/ContactLoggingController.kt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  private val contactLoggingService: ContactLoggingService,
+) {
+  @Operation(summary = "Log contact metrics")
+  @ApiResponses(
+    value = [
+      ApiResponse(responseCode = "200", description = "OK"),
+      ApiResponse(responseCode = "500", description = "Internal server error"),
+    ],
+  )
+  @PreAuthorize("hasRole('ROLE_WORKLOAD_MEASUREMENT') or hasRole('ROLE_WORKLOAD_READ')")
```
Comment (GaReid) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/controller/ContactLoggingController.kt:27]: Do we also need ROLE_MANAGE_A_WORKFORCE_ALLOCATE role here?

Comment (sambailey-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/controller/ContactLoggingController.kt:27]: Good question.. I looked at a few of the other endpoints in here and they mostly have these two.. I'll take a look at allocations and add in

Comment (sambailey-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/controller/ContactLoggingController.kt]: all fixed!

**Review by GaReid — APPROVED** [review]
lgtm

**Pull request merged**